client/test/connection/ssh/index.test.ts

```diff
@@ -0,0 +1,328 @@
+import * as sinon from "sinon";
+import { Client, ClientChannel } from "ssh2";
+import * as fs from "fs";
+import { SSHSession } from "../../../src/connection/ssh";
+import { assertThrowsAsync, getTestFixtureContent } from "../../utils";
+import { assert, expect } from "chai";
+import { StubbedInstance, stubInterface } from "ts-sinon";
+
```

Comment:
There are some small inconsistencies with new lines here. In general, I'm wondering if we can...
 - separate describe sections w/ a new line
 - separate each `it`/`beforeEach`/`afterEach` with a newline

Reply:
No arguments here. Good suggestions. Will make this change.

Reply:
There were a couple more left that I marked down below. Otherwise, this looks good to me